frontend/package.json

```diff
@@ -20,6 +20,7 @@
     "test": "jest",
     "debug-test": "node --inspect-brk node_modules/.bin/jest --runInBand",
     "webdriver-update": "webdriver-manager update",
+    "webdriver-update-protractor": "webdriver-manager update --versions.chrome=76.0.3809.0",
```

Comment:
I wouldn't do it this way. This can break people running locally with a different Chrome version. If we really want this, it should be in the test-prow script.

We should try to find out why the wrong version is installed in the first place, however, likely a dependency problem.